README.md

```diff
@@ -63,5 +63,29 @@ make create-lambda-ebs-backup
 ### Tagging Volumes for backup
 
 In order to backup a volume, you must opt-in by setting a tag key:value pair on
-the volume. By default, this is `lambda-ebs-backup:true`. This tells the lambda
-function that we are to create a snapshot of the volume.
+the volume. By default, this is `lambda-ebs-backup/backup:true`. This tells the
+lambda function that we are to create a snapshot of the volume.
+
+
+#### Controlling how many Volume Snapshots are kept
+
+By default, only 2 volume snapshots are kept. You can override this globally
+by setting the environment variable `DEFAULT_MAX_KEEP_SNAPSHOTS` in the
+cloudformation for the lambda function. Alternatively, you can control this on
+a volume by volume basis by setting the following tag key:value pair on the
+volume: `lambda-ebs-backup/max-keep-snapshots`.
```

Comment:
If both are defined, I assume the tag takes precedence over the environment variable?https://github.com/C2FO/lambda-ebs-backup/pull/2/files#diff-1de92dee00ed13c236d39c6583aa8329R64

Reply:
@bksteiny, that is correct. The global default is 2 unless overwritten by the environment variable. You can further change this behavior on a volume by volume basis by the value of that tag to whatever you want.